CMakeLists.txt

```diff
@@ -745,9 +745,9 @@ if(NOT WIN32)
   # Add common library search paths so executables and libraries can load and run
   # without LD_LIBRARY_PATH being set.
   link_libraries(
-    "-Wl,-rpath,'$ORIGIN'"
-    "-Wl,-rpath,'$ORIGIN/../lib64'"
-    "-Wl,-rpath,'$ORIGIN/../lib'"
+    "-Wl,-rpath,'$$ORIGIN'"
```

Comment:
I'm curious about this change. CMake's documentation shows the use of a single `$`, but I do see that we now wrap with a secondary set of `''` which I wonder if that makes a difference. What system are you testing on just to get an idea of where this was failing.

Reply:
Amz Linux 2023 (Fedora based)

Reply:
This might actually be a good time to look at using the `CMAKE_BUILD_RPATH`, `CMAKE_INSTALL_RPATH`, and `CMAKE_BUILD_RPATH_USE_ORIGIN` flags which were not fully working when CUTLASS started 😄